src/main/java/org/commcare/formplayer/beans/menus/BaseResponseBean.java

```diff
@@ -8,13 +8,15 @@
  * Base class for responses being sent to the front end. Params are:
  * title - self explanatory
  * notification - A message String and error boolean to be displayed by frontend
+ * sholdAutoSubmit - A boolean to indicate if the frontend should auto-submit the form
  * clearSession - True if the front end should redirect to the app home
  *  NOTE: clearSession causes all other displayables to be disregarded
  */
 public class BaseResponseBean extends LocationRelevantResponseBean {
     protected NotificationMessage notification;
     protected String title;
     protected boolean clearSession;
+    protected boolean shouldAutoSubmit;
```

Comment:
I am having trouble having this attributed be returned in the final JSON that the front-end gets. I did `gradle clean, deleted formplayer.jar and built formplayer again. But still, the final JSON didn't have this attribute. Is there any other serializer where I need to add this? 

FWIW just to make sure I compiled correctly, when I renamed `clearSession` var in all palces of the code and the final JSON reflected that change. So, I am not sure what else am I missing to get `shouldAutoSubmit` passed into final formplayer JSON.

Any clues? @ctsims @calellowitz